easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1_icu_sha_changed.patch

```diff
@@ -0,0 +1,14 @@
+# ICU SHA256 changed upstream see:
+# https://github.com/tensorflow/tensorflow/issues/24135
+diff -ru tensorflow-1.13.1.orig/third_party/icu/workspace.bzl tensorflow-1.13.1/third_party/icu/workspace.bzl
+--- tensorflow-1.13.1.orig/third_party/icu/workspace.bzl	2019-02-25 21:37:04.000000000 +0100
++++ tensorflow-1.13.1/third_party/icu/workspace.bzl	2021-07-01 23:02:32.266223519 +0200
+@@ -11,7 +11,7 @@
+     third_party_http_archive(
+         name = "icu",
+         strip_prefix = "icu-release-62-1",
+-        sha256 = "e15ffd84606323cbad5515bf9ecdf8061cc3bf80fb883b9e6aa162e485aa9761",
++        sha256 = "86b85fbf1b251d7a658de86ce5a0c8f34151027cc60b01e1b76f167379acf181",
+         urls = [
+             "https://mirror.bazel.build/github.com/unicode-org/icu/archive/release-62-1.tar.gz",
```

Comment:
I'd suggest to rather fix the URL: 
- old:  https://mirror.bazel.build/github.com/unicode-org/icu/archive/release-62-1.tar.gz
- new: http://mirror.tensorflow.org/github.com/unicode-org/icu/archive/release-62-1.tar.gz

Probably even with a sed instead of the patch for https://mirror.bazel.build -> http://mirror.tensorflow.org

See https://github.com/easybuilders/easybuild-easyconfigs/issues/11253

Reply:
I disagree with it, There are 2 URL given, the second one is ICU's github page. if I fix the bazel URL, then we have mismatching checksums between the the 2 URLs.

Reply:
That is expected. The point that the ICU github page returns sources with another checksum is a bug on the TF side. They expect the sources with resolved LSF files. Giving them other sources a) changes current/old behavior and b) may break the build.

-> We always had mismatching checksums and that is fine as long as the first URL points to an existing, correct source.

Reply:
fixed